src/components/Icon/LoadingIcon/LoadingIcon.less

```diff
@@ -0,0 +1,44 @@
+.lucid-LoadingIcon {
+	// simple rotation in IE11 because IE11 doesn't support svg animations
+	-ms-animation: rotate-animation linear infinite;
+
+	circle {
+		fill: none;
+		stroke-width: 12;
+		stroke-linecap: round;
+	}
+
+	&-spinner {
+		transform-origin: center;
+		animation: dash-animation linear infinite;
+		stroke: @color-primary;
+		stroke-dasharray: 150px 100px;
+	}
+
+	&-circle {
+		stroke: @color-gray;
+	}
+
+}
+
+@keyframes rotate-animation {
```

Comment:
Probably not a big deal, but shouldn't we prefix the keyframe with the component name as well?